python/ray/experimental/async_plasma.py

```diff
@@ -36,19 +33,30 @@ def close(self):
                 fut.cancel()
 
     def _complete_future(self, futures: List[asyncio.Future], ray_object_id):
+        logger.debug(
+            "Completing plasma futures for object id {}".format(ray_object_id))
+
         obj = ray.get(ray_object_id)
         for fut in futures:
```

Comment:
Let's mark this as todo here so in this future it can be easier to track down. i don't think this edge case is in scope for this PR though. Because the case requires someone to `as_future` an object id, wait on it and somehow call as_future on the same object id at the same time.